src/test/scala/invertible/composed.scala

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright 2014 - 2016 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package invertible
+
+import invertible._
+import matryoshka._, TraverseT._, Recursive.ops._, Corecursive.ops._
```

Comment:
`Corecursive.ops._` is empty. All the Corecursive stuff (as well as things like `hylo`) are on the IdOps, which is manually-written and imported with `matryoshka._`.